src/driver.c

```diff
@@ -34,15 +34,18 @@
 #include <sys/wait.h>
 #include <unistd.h>
 
-#define DEVICE_DRIVER_COUNT 2
+#define DEVICE_DRIVER_COUNT 2 
 const device_driver_type* device_drivers[DEVICE_DRIVER_COUNT] = {
     &xreal_driver,
     &viture_driver
 };
 
 #define EVENT_SIZE (sizeof(struct inotify_event) + NAME_MAX + 1)
-#define MT_RECENTER_SCREEN 2
-#define MT_RESET_CALIBRATION 3
+#define MT_SINGLE_CLICK 1
+#define MT_DOUBLE_CLICK 2
+#define MT_RECENTER_SCREEN 3
```

Comment:
Hmm... I think this is problematic, though I'm not sure the solution. I'd like to keep recenter/recalibrate at 2 and 3 taps, respectively, for a couple reasons: multi-tap is difficult to get right for some, so going beyond double-tap for something you may use a lot can add a bit of pain, and all my documentation, UI, and videos talk about double-tap for recenter.

The other issue I see with this is having something use single-tap. False-positives happen sometimes during head movements, so I feel hesitant adding something that triggers after just one tap.